primitives/hp-proof-of-existence/src/lib.rs

```diff
@@ -94,3 +95,25 @@ impl sp_inherents::InherentDataProvider for InherentDataProvider {
 pub trait OnProofVerified {
     fn on_proof_verified(pubs_hash: H256);
 }
+
+/// Wrapper around u32 for MaxStorageAttestations
+#[derive(Eq, PartialEq)]
+pub struct MaxStorageAttestations(pub u32);
```

Comment:
Why don't use `NonZeroU32` here?

Reply:
Actually, I've enforced to be at least 1, otherwise it's the same effect as 0 with the current implementation